layers/auto-completion/packages.el

```diff
@@ -263,18 +264,6 @@
                                                                eshell-mode-hook)))
     :config
     (progn
```

Comment:
Yeah but' it's not multiline anymore? Or do you also want the `(progn …` for single commands?